Roadmap.md

```diff
@@ -44,28 +44,30 @@ https://github.com/ry/deno/master/testing.js
 * Threat model:
   * Modifiying/deleting local files
   * Leaking private information
-* By default:
+* Disallowed default:
     * No network access
```

Comment:
Perhaps you might want to remove "No" to avoid double negative?

Reply:
thanks - fixed